### ISSUE TITLE
Cron not running

### ISSUE THREAD

**Issue by haynes11** [domainmod/domainmod#48]
Hi,

I don't believe the cron job is running. Can someone confirm to me what command they are using to run their cron job?

I'm running mine on cpanel/WHM.

Best,
Jerome

**Comment by lucentminds** [comment]
Most cpanels are on a linux system so you might be able to use curl. I personally wrote a script in php that runs in cli. All it does is just hit a url with some parameters. Then I created a cron job that periodically calls `php my_php_script.php`.

**Comment by haynes11** [comment]
This looks to be failing so I was wondering if it is correct? It's set to run every 10 minutes

/usr/local/bin/php -q /home/techsu/public_html/domaintracker/cron.php

**Comment by lucentminds** [comment]
Check your php error logs to see if you're getting an error. If everything is working correctly and domainmod has no notifications for you, then you should get no feedback from domainmod.

If you just want to make sure cron is operating correctly you can turn on debug logging in "_domainmod/admin/settings/_". It will create a log entry each time cron.php is run.

**Comment by haynes11** [comment]
Cpanel cron is e-mailing me saying; Could not open input file: /home/techsu/public_html/domaintracker/cron.php

**Comment by lucentminds** [comment]
Try `/usr/local/bin/php -f /home/techsu/public_html/domaintracker/cron.php`

**Comment by haynes11** [comment]
Still getting the same problem

**Comment by lucentminds** [comment]
Sounds like the path `/home/techsu/public_html/domaintracker/cron.php` is bad somehow. Here are some things to try...

* Double check the path to the `cron.php` file. Make sure you can hit it in your browser.
* If your sure the path is correct, you may want to contact your hosting support. They may be able to help you figure out why php can't find the file on that path.
* Use __curl__. Set up a cron job to hit the url of your `cron.php`. Something like `curl -s 'https://example.com/domaintracker/cron.php' > /dev/null`.

**Comment by haynes11** [comment]
Turned out to be the path... (Forgot that the vhost is actually pointed to a directory further down) So /domaintracker/domainmod/cron.php -  looks like I've finally got it working. Next issue is I can't pull data through from namecheap I just get "

error | Unable to get domain list
-- | --

"

as the error in debug log. API key is correct. Unknown what to put in API secret (Tried the API key in both and still didn't work) Is this a broken feature?

Best,
J
